build/mac/download_hermetic_xcode.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env vpython3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Brave copyright?

Reply:
this is the copyright we are using on python scripts for some reason, at least the ones I saw